web-marketplace/src/components/organisms/BuyCreditsModal/hooks/useFetchUserBalance.tsx

```diff
@@ -5,16 +5,15 @@ import { useLedger } from 'ledger';
 import { getAllBalancesQuery } from 'lib/queries/react-query/cosmos/bank/getAllBalancesQuery/getAllBalancesQuery';
```

Comment:
we could actually move this hook to the new  /pages/BuyCredits/hooks and delete BuyCreditsModal entirely since it's not used anymore